### ISSUE TITLE
Android 13 permissions 

### ISSUE THREAD

**Issue by Serroda** [diachedelic/capacitor-blob-writer#64]
`write_blob` throws this error "Unable to do file operation ,user denied permission request".
My AndroidManifest has `READ_MEDIA_*` permissions, `WRITE_READ_STORAGE` and `READ_EXTERNAL_STORAGE`.

I have also enabled the app permissions in the options but it doesn't work

**Comment by Serroda** [comment]
Mmm 

I have realized that the error comes from the `directory` parameter. When you set `Directory.ExternalStorage` doesnt work

Because `It's not accesible on Android 11 or newer.`
https://capacitorjs.com/docs/apis/filesystem#directory

Maybe workaround writting `file://` in `path` param ?

**Comment by diachedelic** [comment]
The relevant code is [here](https://github.com/diachedelic/capacitor-blob-writer/blob/a0c8a2f8eb3f367925a2c22f1102904911b59ac3/android/src/main/java/com/equimaps/capacitorblobwriter/BlobWriterServer.java#L110), see what you can find out.

**Comment by Serroda** [comment]
If you put the absolute URL `file:///storage/emulated/0/` works but now the file is empty with 0 bytes. I will continue to investigate

**Comment by Serroda** [comment]
Ok i solved the problem. When I was testing i have in my `capacitor.config.ts` this configuration:

```
const config: CapacitorConfig = {
  appId: 'com.example.app',
  appName: 'Example',
  webDir: 'dist',
  plugins: {
    CapacitorHttp: {
      enabled: true
    },
  }
};

```
https://capacitorjs.com/docs/apis/http#configuration

`Enable the patching of fetch and XMLHttpRequest to use native libraries instead.`

When is enable  content-length header is `0`

![Captura desde 2023-06-03 13-37-59](https://github.com/diachedelic/capacitor-blob-writer/assets/35766979/a9bad35c-1dc1-4086-8ef4-35a767370605)

Just put  `enabled: false`

**Comment by diachedelic** [comment]
Thank you. I have updated the README with a warning.

**Comment by diachedelic** [comment]
I just released v1.1.11. It is compatible with CapacitorHttp's fetch patching feature.
